woocommerce/woocommerce-gutenberg-products-block
# Revert "fix: accessibility of search products block (#5691)"

This reverts commit cd5c5dedf731aa5a2d6938ecd60a4f55b9b55181.

Reasoning is it has minor impact but could cause blocks invalidation for already existing blocks as caught by E2E and missed in testing. As we have plans for bigger rewrite of this component in the future this change is not worth it. The underlaying issue it was addressing is partially covered as the currently used php rendering of the block has correct aria-label set and block should be accessible on the frontend.

At the same time I believe #1671 can remain closed as it was tested and confirmed that PHP rendering on the frontend does produce correct aria-lable as mentioned but this was fixed/correct before the change that #5691 introdcued.




Reverts #5691




#### Accessibility

### Testing

#### Automated Tests

### Manual Testing

How to test the changes in this Pull Request:

1. E2E tests should pass and there should be no errors for product-search block

### User Facing Testing

Follow the steps from #5691 



### Changelog

> ⚠️ Removes changelog entry from #5691
